pytest/Pipfile

```diff
@@ -0,0 +1,24 @@
+[[source]]
```

Comment:
Bad merge? I’ve deleted Pipfile a few commits back since we already have requirements.txt and it’s best to have one source where dependencies are listed.